### PR TITLE
Use PNG country flags

### PR DESCRIPTION
**Pull request by Ken-Lo**
Changes the country flags in the footer to use PNG format. Reduces the total image download size from 241KB to 8.6KB.

flag selector

### CODE REVIEW

**Comment by Ken-Lo** [comment]
Hey @harrison 
Any chance of a review?
Thanks!

**Comment by harrison** [comment]
Codes :+1:
The uk flag looks a bit jagged, worth finding a good icon pack with small sizes? Resizing a big icon with detail will always looks a bit crap
@anothertompetty @swillis you have any good ones floating around?

**Comment by swillis** [comment]
@Ken-Lo & @Ken-Lo I do indeed have a little collection of flags! I have updated the PNGs and a little of the CSS. They look much cleaner now. One of the issues was that the flag in the footer indicating the selected country was slightly larger than the others. Fine if it is an SVG, but upscaling the little PNG was proving too much! :smile:

**Comment by harrison** [comment]
@swillis oh sweet! :)

**Comment by Ken-Lo** [comment]
@swillis Nice one - thanks! Going to merge this now, when we get some time we should also update the footer in the guides (if we think there should even be country flags there).

File: app/components/flag/flag.js [hunk 1]
```diff
@@ -1,39 +1,18 @@
...
-    },
-  };
+    country: PropTypes.string.isRequired,
+  }
 
   render() {
-    const countryCode = this.props.country.toString().toLowerCase();
-    const FlagIcon = countryMap[countryCode];
+    const countryCode = this.props.country.toString();
+    const flagPath = `/images/flags/${countryCode}-flag-icon@2x.png`;
```
Comment (Ken-Lo) [inline app/components/flag/flag.js]: @harrison Quick q before merging: is it safe to use ES6 interpolation here?

Comment (harrison) [inline app/components/flag/flag.js]: :+1: it all get's transpiled to es5 before running in the browser

**Pull request merged**